samples/sensor/nine_degrees_freedom/src/main.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * 08/23/2018	Jose Sune Cofreros Jr. 	Created nine degrees of freedom 
+ *					sensor suit using MPU6050 and 
+ *					HMC5883L.
+ *
+ * 08/26/2018	Jose Sune Cofreros Jr.	Verified on Nucleo STM32L476RG
+ *					and Intel Quark D2000.
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <sensor.h>
+#include <stdio.h>
+
+void main(void)
+{
+	struct device *dev1 = device_get_binding("MPU6050");
+	struct device *dev2 = device_get_binding("HMC5883L");
+
```

Comment:
you should check dev1 and dev2 are not NULL

Reply:
Done.